### ISSUE TITLE
idempotency travis testing is not right

### ISSUE THREAD

**Issue by martbhell** [CSC-IT-Center-for-Science/ansible-role-squid#13]
While debugging some idempotency issues in another repo I found that there's a one too many pipes in some repos:

```
roles/ansible-role-squid/tests/test-in-docker-image.sh:    ansible-playbook -i ${ANSIBLE_INVENTORY} ${ANSIBLE_PLAYBOOk} ${ANSIBLE_LOG_LEVEL} --connection=local ${SUDO_OPTION} ${ANSIBLE_EXTRA_VARS} || grep -q 'changed=0.*failed=0' && (echo 'Idempotence test: pass' ) || (echo 'Idempotence test: fail' && exit 1)
```

has 

```
ansible-command || grep -q 'changed=0.*failed=0' && (echo 'Idempotence test: pass' ) || (echo 'Idempotence test: fail' && exit 1)
```

Which should probably have one fewer pipe before the "grep -q ..." bit.

```
ansible-command | grep -q 'changed=0.*failed=0' && (echo 'Idempotence test: pass' ) || (echo 'Idempotence test: fail' && exit 1)
```

**Comment by VilleS1** [comment]
Fixed in a7be3399cdb096064b11bd56a924e0130058005a

**Comment by martbhell** [comment]
Thanks for all of these :) travis is also passing for this role after the change :)
